styles/_markdown.css

```
@@ -14,19 +14,29 @@
 }
 
 .md-ol > .md-li p::before {
-  content: counter(li) ".";
+  content: counter(li) "•";
```

This is the placeholder character for an ordered list.. should probably be changed back to `1.`, `2.`,.. instead of `1•`, `2•`,...

```suggestion
  content: counter(li) ".";
```

---

Yes, I will do so!